lib/Github/Api/CurrentUser.php

```diff
@@ -104,7 +104,6 @@ public function teams()
      * @param  string $type      role in the repository
      * @param  string $sort      sort by
      * @param  string $direction direction of sort, ask or desc
-
```

Comment:
I was going to send a pull request later to standardise the annotation separation.

I want to keep the PRs atomic.